vision_agent/tools/tools.py

```diff
@@ -198,14 +198,17 @@ def extract_frames(
 
 def ocr(image: np.ndarray) -> List[Dict[str, Any]]:
     """'ocr' extracts text from an image. It returns a list of detected text, bounding
-    boxes, and confidence scores. The results are sorted from top-left to bottom right
+    boxes with normalized coordinates, and confidence scores. The results are sorted
+    from top-left to bottom right. Each element in the returned list represents a
+    contiguous text region. Text on separate lines is considered separate regions and
```

Comment:
`"Text on separate lines or text with more than a tab spacing between words is considered separate regions"`

Reply:
Reverted this change